gui/tests/example.spec.ts

```diff
@@ -0,0 +1,23 @@
+import { test, expect } from '@playwright/test';
+
+// test('should contain the word "Welcome" on the homepage', async ({ page }) => {
```

Comment:
This function can probably be removed instead of just left commented.

Reply:
I removed that function.